### PR TITLE
[v2.0.3] Swagger Configuration for Response Datastore API documentation

### PR DESCRIPTION
**Pull request by chiranjibi009**
This PR contains issue fixes for issue #3125 (Response Datastore) .
Added test to create openapi.json under /fda-mystudies/documentation/API/response-datastore

API Documentation URL: http://<HOST>:<PORT>/response-datastore/swagger-ui.html# or copy the content of this file into [Swagger Editor](https://editor.swagger.io/)

### CODE REVIEW

**Comment by madhurya-btc** [comment]
@googlebot I consent.

File: response-datastore/response-server-service/src/main/java/com/google/cloud/healthcare/fdamystudies/FdaMystudiesResponseServerApplication.java [hunk 1]
```diff
@@ -10,7 +10,9 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.scheduling.annotation.EnableScheduling;
```
Comment (moschetti) [inline response-datastore/response-server-service/src/main/java/com/google/cloud/healthcare/fdamystudies/FdaMystudiesResponseServerApplication.java:13]: update copyright to 2021

File: response-datastore/response-server-service/src/main/java/com/google/cloud/healthcare/fdamystudies/controller/ProcessActivityResponseController.java [hunk 2]
```diff
@@ -92,6 +95,7 @@
   private static final Logger logger =
       LoggerFactory.getLogger(ProcessActivityResponseController.class);
 
+  @ApiOperation(value = "Process activity response for participant and store in cloud fire store")
```
Comment (moschetti) [inline response-datastore/response-server-service/src/main/java/com/google/cloud/healthcare/fdamystudies/controller/ProcessActivityResponseController.java:98]: `Cloud Firestore`

File: response-datastore/response-server-service/src/main/java/com/google/cloud/healthcare/fdamystudies/controller/ProcessActivityResponseController.java [hunk 3]
```diff
@@ -388,6 +392,7 @@
     }
   }
 
+  @ApiOperation(value = "Get activity response data for participant from cloud fire store")
```
Comment (moschetti) [inline response-datastore/response-server-service/src/main/java/com/google/cloud/healthcare/fdamystudies/controller/ProcessActivityResponseController.java:395]: `Cloud Firestore`

File: response-datastore/response-server-service/src/test/java/com/google/cloud/healthcare/fdamystudies/SwaggerGeneratorTest.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Google LLC
```
Comment (moschetti) [inline response-datastore/response-server-service/src/test/java/com/google/cloud/healthcare/fdamystudies/SwaggerGeneratorTest.java:2]: 2021

**Review by moschetti — APPROVED** [review]

**Comment by chiranjibi009** [comment]
Hi moschetti, The PR status is changed after the base branch got merged with develop branch, Please reapprove this PR.

File: response-datastore/response-server-service/src/main/java/com/google/cloud/healthcare/fdamystudies/controller/StudyMetadataController.java [hunk 5]
```diff
@@ -41,6 +48,9 @@
 
   private static final Logger logger = LoggerFactory.getLogger(StudyMetadataController.class);
 
+  @ApiOperation(
+      value =
+          "Add or update study metadata in response datastore when a study is published from study builder")
```
[the remark this answers is not in the document]
Comment (chiranjibi009) [reply response-datastore/response-server-service/src/main/java/com/google/cloud/healthcare/fdamystudies/controller/StudyMetadataController.java]: code formatted

**Review by moschetti — APPROVED** [review]

**Pull request merged**